processing/src/main/java/org/apache/druid/query/Query.java

```diff
@@ -129,6 +130,8 @@ default QueryContext getQueryContext()
 
   boolean getContextBoolean(String key, boolean defaultValue);
 
+  HumanReadableBytes getContextHumanReadableBytes(String key, HumanReadableBytes defaultValue);
```

Comment:
Since `Query` is marked as an `ExtensionPoint` I think we should introduce a default implementation of this method to help users who may have extended this class upgrade to this version of Druid without having to make a modification to their extension. The default implementation could be something like

```
default HumanReadableBytes getContextHumanReadableBytes(String key, HumanReadableBytes defaultValue) {
  if (getQueryContext != null) {
    return getQueryContext().getAsHumanReadableBytes(key, defaultValue);
  }
}
```

Reply:
Added this default basically word for word. Added a return of the defaultValue if the context doesn't exist